src/manage_nsfs/health.js

```diff
@@ -378,6 +384,27 @@ class NSFSHealth {
         };
     }
 
+    /**
+ * get_config_file_data_or_error_object return an object containing config_data or err_obj if error occurred
+ * @param {string} name
+ * @param {string} owner_account
+ * @returns {Promise<object>}
+ */
+    async get_account_owner_context(name, owner_account) {
+        let account_fs_context;
+        try {
+            if (owner_account) {
+                const owner_account_data = await get_bucket_owner_account_by_id(this.config_fs, owner_account);
+                account_fs_context = await native_fs_utils.get_fs_context(owner_account_data.nsfs_account_config,
+                                        owner_account_data.nsfs_account_config.fs_backend);
+            }
```

Comment:
why not throw an error from here? instead of checking if it's undefined outside and then throw an error